StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -213,7 +205,84 @@ class EmbeddedPaymentMethodsView: UIView {
     }
 
     func didTapAccessoryButton() {
-        // TODO(porter)
+        guard let savedPaymentMethod = savedPaymentMethods.first else {
+            stpAssertionFailure("Tapped on accessory button but there are no saved payment methods")
+            return
+        }
+        delegate?.presentSavedPaymentMethods(selectedSavedPaymentMethod: savedPaymentMethod)
+    }
+
+    func updateSavedPaymentMethods(_ savedPaymentMethods: [STPPaymentMethod],
```

Comment:
I think this function should be `updateSavedPaymentMethodRow(savedPaymentMethod: STPPaymentMethod?, ...)`

Reply:
If we changed this function to only pass in a single savedPaymentMethod, I'm not quite sure how we'd make it work if we update to multiple items in the list.  For example:
1. You have one CBC card and one regular card, then present Embedded Payment Element.
2. Tap view more on the SPM to reveal the `VerticalSavedPaymentMethodsViewController`
3. Tap edit
4. Update first card to switch the cbc
5. Remove the regular card.
6. Hit back to reveal the `EmbeddedPaymentMethodsView`

At this point, we've updated one card's CBC, and removed another card, and then as soon as we hit 'back', the code calls "VerticalSavedPaymentMethodsViewControllerDelegate.didComplete" with `latestPaymentMethods: [STPPaymentMethods]` which contains both of those changes.

Reply:
I'm not understanding the example but I'm not seeing where EmbeddedPaymentMethodsView uses savedPaymentMethods in the code and I wouldn't expect it to need more than the one optional saved payment method that it displays.

Reply:
Sorry, I think I was looking at a function that belonged to EmbeddedPaymentElement, not EmbeddedPaymentMethodsView.

Yes, we can get away with only passing in a single savedPaymentMethod to this view!